diff --git a/src/dnsmasq.c b/src/dnsmasq.c
index 19a6428..9493d74 100644
--- a/src/dnsmasq.c
+++ b/src/dnsmasq.c
@@ -134,7 +134,7 @@ int main (int argc, char **argv)
      open to /dev/null. Normally we'll be started with 0, 1 and 2 open, 
      but it's not guaranteed. By opening /dev/null three times, we 
      ensure that we're not using those fds for real stuff. */
-  for (i = 0; i < max_fd; i++)
+  for (i = 0; i < 1024; i++)
     if (i != STDOUT_FILENO && i != STDERR_FILENO && i != STDIN_FILENO)
       close(i);
     else
